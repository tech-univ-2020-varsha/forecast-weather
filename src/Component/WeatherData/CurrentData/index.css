.currentData{
    display: flex;
    flex-direction: column;
    justify-content: center;
    padding: 1em;
    margin: 1em;
    background-color: rgb(234, 244, 246);
    border:solid gray;
    width: 60%;
    
    height: 10%;
}

.dataItems{
    display: flex;
    padding: 1em;
    margin: 1em;
    border:solid  gray;
    justify-items: center;
    align-items: center;
    text-align: center;
    color: gray;
    font-weight: bold;
}