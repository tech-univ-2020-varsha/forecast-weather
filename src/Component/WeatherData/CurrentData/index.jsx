import React from 'react';
import './index.css';

const CurrentData = ({ temp, pressure, humidity }) => (
  <div className="currentData">
    <div className="dataItems">
      TEMPERATURE:
      {' '}
      {temp}
    </div>
    <div className="dataItems">
      PRESSURE:
      {' '}
      {pressure}
    </div>
    <div className="dataItems">
      HUMIDITY:
      {' '}
      {humidity}
    </div>
  </div>
);


export default CurrentData;
