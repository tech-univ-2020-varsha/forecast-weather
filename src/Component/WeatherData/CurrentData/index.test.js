import { render } from '@testing-library/react';
import React from 'react';
import CurrentData from '.';

describe('the CurrentData function', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<CurrentData temp={898} pressure={9898} humidity={9998} />);
    expect(asFragment).toMatchSnapshot();
  });
});
