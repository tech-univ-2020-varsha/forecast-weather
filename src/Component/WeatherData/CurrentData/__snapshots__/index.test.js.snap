// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the CurrentData function should render correctly 1`] = `[Function]`;
